src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -14352,7 +14352,7 @@ void PrintHelp (__in_z LPCSTR pszCmdName)
             return;
         }
 #else
-        int err = PAL_InitializeDLL();
+        int err = PAL_InitializeDLL(TRUE);
```

Comment:
Instead of the bool parameter, it seem it would be nicer to create a new PAL_InitializeDLLWithEH (or a similar name like that). It would be easier to understand what the call actually means.